src/ophyd_async/core/signal.py

```diff
@@ -2,9 +2,20 @@
 
 import asyncio
 import functools
-from typing import AsyncGenerator, Callable, Dict, Generic, Optional, Union
+from typing import (
+    Any,
+    AsyncGenerator,
+    Awaitable,
+    Callable,
+    Dict,
+    Generic,
+    Mapping,
+    Optional,
+    Union,
+)
 
 from bluesky.protocols import (
+    Configurable,
```

Comment:
Since #219 is merged, please can you update this to use `ophyd_async.protocols.AsyncReadable` and `AsyncConfigurable`